vector/agent/namespaced/resources/vector.yaml

```diff
@@ -0,0 +1,144 @@
+data_dir: /vector-data-dir
+sources:
+  kubernetes_logs:
+    type: kubernetes_logs
+
+  systemd_logs:
+    type: journald
+    current_boot_only: true
+    journal_directory: /var/log/journal
+
+transforms:
+  drop_fluentbit:
```

Comment:
this reads like we are dropping fluentbit logs, while in reality we are dropping any exluded logs (that happen to still use the fluentbit annotation). I'd call this `drop_excluded` instead